src/__tests__/to-have-prop.js

```diff
@@ -30,4 +30,7 @@ test('.toHaveProp', () => {
     expect(queryByTestId('text')).not.toHaveProp('allowFontScaling', false),
   ).toThrowError();
   expect(() => expect(queryByTestId('text')).toHaveProp('style')).toThrowError();
+  expect(() =>
+    expect(getByText('text')).toHaveProp('allowFontScaling', 'wrongValue'),
+  ).toThrowError();
```

Comment:
As far as I don't prefer get|query}ByTestId queries, it seems that for consistency and readability it would be better to use it here.

Reply:
it's just for the example, any reason it should behave this way if using getByText here?

Reply:
`getByText('text')` vs `queryByTestId('text')` is just a stylistic issue here, as for current setup, both elements return exactly the same element. 

Optimally, we should be using the same queries, as we encourage our users (`ByText`, `ByHintText`, `ByRole`, etc) everywhere in our tests when possible. But since surrounding code uses `queryByTestId`, the second best solution seems to be consistent with surrounding code to reduce cognitive load for reading user.

Be aware that this is a bit nitpicking :-)

Reply:
i understand, in the current setup it's correct, the returned element of both accessors is the same,
but this fix is for an issue that pops only when using the 7.0.2 version of `testing-library/react-native`, in which the returned elements are not the same, so only the `getByTest` func will fail the test (without the fix). that was the logic behind using `getByTest`.
but no problem to change it to queryByTestId, tell me what you think is best